microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -236,61 +209,65 @@ chrome.tabs.onActivated.addListener(async ({ tabId }) => {
 
 
 <!-- ------------------------------ -->
 #### Opening the extension in the sidebar
 
-<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#open -->
-
-There are several ways to open the extension in the sidebar:
+There are several ways for the user to open the extension in the sidebar:
 
 
 <!-- ---------- -->
-*  Click the **Open in sidebar** icon (![Open in sidebar icon](./sidebar-images/open-in-sidebar-icon.png)), which is displayed next to the extension's name in the Extensions hub:
+###### By clicking an icon
+
+Users can click the **Open in sidebar** icon (![Open in sidebar icon](./sidebar-images/open-in-sidebar-icon.png)), which is displayed next to the extension's name in the Extensions hub:
+
+![Sidebar dialog](./sidebar-images/sidebar-dialog.png)
 
-   ![Sidebar dialog](./sidebar-images/sidebar-dialog.png)
+Or, users can click the extension's custom icon in the toolbar, if it's enabled.  In this example, the extension's custom icon is a circle (![Extension's custom icon](./sidebar-images/custom-icon-for-demo-extension.png)):
+
+![Clicking the extension's icon in the toolbar](./sidebar-images/left-click-toolbar-icon.png)
 
 
 <!-- ---------- -->
-*  Right-click the extension's icon in the toolbar, and then select **Open in sidebar** or **Close sidebar**:
+###### By right-clicking the extension's icon
+
+Users can right-click the extension's icon in the toolbar, and then select **Open in sidebar** or **Close sidebar**:
 
-   ![Right-clicking the shortcut on the toolbar to open the extension](./sidebar-images/toolbar-right-click-shortcut-open.png)
+![Right-clicking the shortcut on the toolbar to open the extension](./sidebar-images/toolbar-right-click-shortcut-open.png)
 
-   ![Right-clicking the shortcut on the toolbar to close the extension](./sidebar-images/toolbar-right-click-shortcut-close.png)
+![Right-clicking the shortcut on the toolbar to close the extension](./sidebar-images/toolbar-right-click-shortcut-close.png)
 
-   The extension's icon appears in the toolbar if the user has clicked the **Show in toolbar** (![Show in toolbar icon](./sidebar-images/show-in-toolbar-icon.png)) icon next to the extension's name in the Extensions hub.
+The extension's icon appears in the toolbar if the user has clicked the **Show in toolbar** (![Show in toolbar icon](./sidebar-images/show-in-toolbar-icon.png)) icon next to the extension's name in the Extensions hub.
 
 
 <!-- ---------- -->
-*  Left-click the extension's custom icon in the toolbar, if it's enabled.  In this example, the extension's custom icon is a circle (![Extension's custom icon](./sidebar-images/custom-icon-for-demo-extension.png)):
+###### By pressing a keyboard shortcut
 
-   ![Left-clicking the extension's icon in the toolbar](./sidebar-images/left-click-toolbar-icon.png)
+Users can press a keyboard shortcut, if the action command is enabled and the action icon is enabled.
```

Comment:
"action icon is enabled" is slightly incomplete. We may want to elaborate it to "action icon is enabled to open the sidebar".

Reply:
Users can press a keyboard shortcut, if the action command is enabled and the action icon is enabled to open the sidebar.